Assets/MixedRealityToolkit/_Core/Utilities/CameraCache.cs

```diff
@@ -16,7 +16,48 @@ public static class CameraCache
         /// <summary>
         /// Returns a cached reference to the main camera and uses Camera.main if it hasn't been cached yet.
         /// </summary>
-        public static Camera Main => cachedCamera == null ? Refresh(Camera.main) : cachedCamera;
+        public static Camera Main
+        {
+            get
+            {
+                var mainCamera = cachedCamera == null ? Refresh(Camera.main) : cachedCamera;
+
+                if (mainCamera == null)
+                {
+                    // No camera in the scene tagged as main. Let's search the scene for a GameObject named Main Camera
+                    var cameras = Object.FindObjectsOfType<Camera>();
+
+                    switch (cameras.Length)
+                    {
+                        case 0:
+                            return null;
+                        case 1:
+                            mainCamera = Refresh(cameras[0]);
+                            break;
+                        default:
+                            // Search for main camera by name.
+                            for (int i = 0; i < cameras.Length; i++)
+                            {
+                                if (cameras[i].name == "Main Camera")
+                                {
+                                    mainCamera = Refresh(cameras[i]);
+                                    break;
+                                }
+                            }
+
+                            // If we still didn't find it, just set the first camera.
+                            if (mainCamera == null)
+                            {
+                                mainCamera = Refresh(cameras[0]);
```

Comment:
Should also Debug.LogWarning to show the first camera has been set as main due to no "Main Camera" found

(Odd couldn't edit code in GitHub here?)

Reply:
It will still throw the assert about the missing main camera tag. I was actually wondering if we should just add it for them?

Reply:
Can't add it for them, in case the primary camera is not their main.
Best we can do is warn them to check.